src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -61,11 +61,14 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
       .flatTap(_ => incMetrics(itemsCreated, assets.size)) *> IO.unit
   }
 
+  def isUpdateEmpty(u: AssetUpdate): Boolean = u == TimeSeriesUpdate()
```

Comment:
Make this private and should probably use `===` from https://typelevel.org/cats/typeclasses/eq.html to catch this type error ;)